src/pages/share/index.tsx

```diff
@@ -1,12 +1,146 @@
+import { Radio, TabButton } from '@/components/atoms';
 import Header from '@/components/organisms/Header';
+import ShareListItem from '@/components/organisms/ShareListItem';
 import { type NextPage } from 'next';
+import { useState } from 'react';
+import {
+  Modal,
+  ModalOverlay,
+  ModalBody,
+  ModalContent,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { AngleIcon, PlusIcon } from '@/assets/icons';
+
+type TabValue = 'enroll' | 'proceeding' | 'complete';
+
+interface Tab {
+  label: string;
+  value: TabValue;
+}
+
+type SortValue = 'latest' | 'earliest';
+
+interface Sort {
+  label: string;
+  value: SortValue;
+}
+
+const TABS: Tab[] = [
+  { label: '나눔 신청', value: 'enroll' },
+  { label: '나눔 중', value: 'proceeding' },
+  { label: '나눔 완료', value: 'complete' },
+];
+
+const SORT_TYPES: Sort[] = [
+  { label: '최신순', value: 'latest' },
+  { label: '마감순', value: 'earliest' },
+];
```

Comment:
```suggestion
type TabValue = 'enroll' | 'proceeding' | 'complete';

interface Tab {
  label: string;
  value: TabValue;
}

type SortValue = 'latest' | 'earliest';

interface Sort {
  label: string;
  value: SortValue;
}

const TABS: Tab[] = [
  { label: '나눔 신청', value: 'enroll' },
  { label: '나눔 중', value: 'proceeding' },
  { label: '나눔 완료', value: 'complete' },
];

const SORT_TYPES: Sort[] = [
  { label: '최신순', value: 'latest' },
  { label: '마감순', value: 'earliest' },
];
```

[IMO]
`constants`로 매직스트링들은 관리하거나 `types`로 분리 해도 좋을것 같습니당

Reply:
말씀 주신대로 types로 일부 타입 정의를 분리, 제거 했습니다 🙂